GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -38,7 +38,6 @@ public interface IGitTfsRemote
         string TfsUsername { get; set; }
         string TfsPassword { get; set; }
         IGitRepository Repository { get; set; }
-        [Obsolete("Make this go away")]
```

Comment:
Oh, I remember now. I wanted to make it so that you never have to do `remote.Tfs.BlahBlahBlah()`, but rather do `remote.BlahBlahBlah()`. It's fine to drop the `Obsolete` attribute.